src/core_plugins/kibana/public/home/components/feature_directory.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Synopsis } from './synopsis';
+import {
+  KuiTabs,
+  KuiTab,
+  KuiFlexItem,
+  KuiFlexGrid,
+} from 'ui_framework/components';
+
+const ALL = 'all';
+
+export class FeatureDirectory extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    const tabs = [{
+      id: ALL,
+      name: 'All',
+    }];
+    if (props.directories.some(directory => directory.category === props.directoryCategories.DATA)) {
+      tabs.push({
+        id: props.directoryCategories.DATA,
+        name: 'Explore & Visualize'
+      });
+    }
+    if (props.directories.some(directory => directory.category === props.directoryCategories.ADMIN)) {
```

Comment:
Might be overkill to make these conditional.  I don't think we'd ever not have stuff in ADMIN and DATA.  Up to you.